packages/webapp/src/components/Task/AbandonTask/index.jsx

```diff
@@ -113,7 +115,21 @@ const PureAbandonTask = ({
 
       {hasAssignee && (
         <>
-          <Main style={{ marginBottom: '24px' }}>{t('TASK.ABANDON_TASK_DURATION')}</Main>
+          <div style={{ display: 'flex', flexDirection: 'row', justifyContent: 'space-between' }}>
+            <Main style={{ marginBottom: '24px' }}>{t('TASK.ABANDON_TASK_DURATION')}</Main>
+            <div style={{ marginBottom: '16px' }}>
+              <Label
+                style={{
+                  paddingRight: '10px',
+                  fontSize: '16px',
+                  lineHeight: '20px',
+                  display: 'inline-block',
+                }}
+              >
+                {<Infoi content={t('TASK.ABANDON_TASK_HELPTEXT')} />}
+              </Label>
+            </div>
+          </div>
```

Comment:
This works but there is a much simpler way of doing the same thing.

The 'Main' component has a 'toolTipContent' prop. We can just pass t('TASK.ABANDON_TASK_HELPTEXT') into 'Main' as a prop and get the same result